postgresql-persistent/postgres-delete.yaml

```diff
@@ -0,0 +1,57 @@
+- hosts: localhost
+  gather_facts: False
+  tasks:   
+    - name: Delete a deployment
+      tags:
+        - ddeployment
```

Comment:
typo

Reply:
I just mentioned it as ddeployment because deployment is mentioned in postgres.install.yaml for kind Deployment? So, should I use the same tag name for installing and deleting a deployment, likewise for backups?